### PR TITLE
Add notification warning when commit to SYNC replica fails

### PR DESCRIPTION
**Pull request by Josipmrden**
Up to now, an exception would be thrown if there was a write query on MAIN instance with at least one replica down. It's now upgraded to only show notification that main has committed data but not synced with all replicas.

[master < Task] PR
- [X] Check, and update documentation if necessary
- [ ] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [X] Write a release note here
- [X] Tag someone from docs team in the comments

Release note: [Replication] When committing new changes on MAIN instance, a notification is displayed instead of exception if data has not been synced with all replicas (e.g. in case a replica is down).

### CODE REVIEW

**Comment by Josipmrden** [comment]
@vpavicic tagging for release note

**Comment by vpavicic** [comment]
@Josipmrden Like this?

From this release, when changes are commited on MAIN instance but cannot be synced with all REPLICA instances (for example, a replica is unavailable), a notification is displayed instead of an exepction.

**Comment by Josipmrden** [comment]
> @Josipmrden Like this?
> 
> From this release, when changes are commited on MAIN instance but cannot be synced with all REPLICA instances (for example, a replica is unavailable), a notification is displayed instead of an exepction.

Yes, I like this one

File: src/query/interpreter.cpp [hunk 1]
```diff
@@ -2590,8 +2590,15 @@ void Interpreter::Commit() {
 
   SPDLOG_DEBUG("Finished committing the transaction");
   if (!commit_confirmed_by_all_sync_repplicas) {
-    throw ReplicationException("At least one SYNC replica has not confirmed committing last transaction.");
+    std::vector<Notification> notifications;
+    notifications.reserve(1);
+    notifications.emplace_back(SeverityLevel::WARNING, NotificationCode::REPLICA_PORT_WARNING,
+                               "At least one SYNC replica has not confirmed committing last transaction.");
```
Comment (gitbuda) [inline src/query/interpreter.cpp:2596]: All this is possible as a part of the constructor line.

Comment (gitbuda) [inline src/query/interpreter.cpp:2596]: Expand the notification with the part in which you are trying to suggest user what it should be done.

**Review by gitbuda — CHANGES_REQUESTED** [review]
Some minor suggestions in the inline comments.

**Comment by antoniofilipovic** [comment]
@Josipmrden what is status with this PR

**Comment by Josipmrden** [comment]
> @Josipmrden what is status with this PR

I'll convert this to Draft and we push it to v2.10

**Comment by andrejtonev** [comment]
@Josipmrden should this be merged?

**Pull request closed without merging**